web/src/hooks/queries/useCounter.ts

```diff
@@ -0,0 +1,21 @@
+import { graphql } from "src/graphql";
+import { useQuery } from "@tanstack/react-query";
+import { CounterQuery } from "src/graphql/graphql";
+import { graphqlQueryFnHelper } from "utils/graphqlQueryFnHelper";
+
+const counterQuery = graphql(`
+  query Counter {
+    counter(id: "0") {
```

Comment:
here I would add all the counter fields preemptively, so that we can use this hooks whenever we want to access the counter.